Adds more job relevant items to drobes

Archimus12 opened the pull request:
## About The Pull Request
Adds coloured large scarfs and winter coats to ClothesMate
Adds latex gloves to SciDrobe, ChemDrobe, ViroDrobe and GeneDrobe
Adds nitrile gloves to MediDrobe
Adds science headsets to GeneDrobe and RoboDrobe
Adds medical headsets to ViroDrobe and ChemDrobe
Adds black shoes to ChefDrobe
Adds black wizard robes and black wizard hat to MagiVend
Adds grey backpack, grey satchel, leather satchel and grey duffel bag to CargoDrobe
## Why It's Good For The Game
It makes it so that some job relevant items such as gloves are in drobes which makes them harder to steal, this will make it less likely that a latejoin medical doctor will be unable to find and nitrile gloves. The gloves are added in departments which already have access to them so it won't result in more people having access to the gloves.
## Changelog
:cl:
add: Adds coloured large scarfs and winter coats to ClothesMate and black shoes to ChefDrobe
add: Adds latex gloves to SciDrobe, ChemDrobe and GeneDrobe and nitrile gloves to MediDrobe
add: Adds job headsets to GeneDrobe and RoboDrobe, ViroDrobe and ChemDrobe
add: Adds black wizard robes and black wizard hat to MagiVend
add: Adds grey backpack, grey satchel, leather satchel and grey duffel bag to CargoDrobe
add: Organises Drobes
/:cl:

Archimus12 committed:
Fixed admin custom job slot

Removed "jebs" from custom slot code.
Made cancel button not set job slots to 0.

Archimus12 committed:
Merge branch 'master' into Archimus

Archimus12 committed:
More job related stuff in vending machines

Added coloured large scarfs and winter coats to ClothesMate
Added latex gloves to SciDrobe, ChemDrobe and GeneDrobe
Added science goggles to CciDrobe, ViroDrobe and ChemDrobe
Added medical huds to ViroDrobe and MediDrobe
Added science headsets to GeneDrobe and RoboDrobe
Added medical headsets to ViroDrobe, ChemDrobe and MediDrobe
Added engineering headsets to EngiDrobe and AtmosDrobe
Added black shoes to ChefDrobe
Added black wizard robes and black wizard hat to MagiVend
Added grey backpack, grey satchel and grey duffel bag to CargoDrobe
Added leather satchel to CargoDrobe premium section

Archimus12 committed:
Adds bags to drobes

Adds science bag to SciDrobe
Adds bio bag to ViroDrobe

Archimus12 committed:
Update magivend.dm

Archimus12 committed:
Update wardrobes.dm

leather is not that fancy

Archimus12 committed:
Adds sec hud and headset to SecDrobe

Adds sec hud (not sunglasses) and headset (not bowman) to SecDrobe

Archimus12 committed:
Merge remote-tracking branch 'upstream/master' into Archimus

YakumoChen commented:
I'm going to hate you for this downstream, but proceed

MrMelbert reviewed (commented):
HUDs are printable in the lathe with the prerequisite tech, do they really need to be in the vendor?

Sealed101 commented:
nitriles for sci and gene? 

overall this is nice to have since maps are inconsistent with such equipment being shiftstart in departments

LemonInTheDark commented:
don't like the huds in drobes either

Archimus12 committed:
undid accidental reduction of ph booklet amount

Archimus12 commented:
The idea I had behind it was that often times some maps can lack important round start equipment.
Nitrile gloves are only in med I made a typo in pr, it is now fixed.
For HUDs in drobes I noticed that Robotics has theirs in the drobe.

LemonInTheDark commented:
HUDs not being available shiftstart is intentional. that's why the research is a thing. sidestepping that by just putting them in the drobe seems silly

GuillaumePrata commented:
Roboticist huds are on the vendor because the locker doesn't spawn on any map.
Which is just really weird, but I never realized in the last 4-5 years here sooo...

zxaber commented:
Robotics used to have a locker, but it was replaced something like four years ago.

IIRC, HUD tech is pretty solidly mid-game, possibly mid/late. But every department with HUDs gets them roundstart.

Archimus12 commented:
The departments already have the HUDs shift start so I think it isn't sidestepping science much more than they already do with the HUDs on tables/in lockers.
I can removed the HUDs (and science goggles?) from the pr if no one wants that change as it isn't really my focus with this pr.
Are there any other parts of the pr that I should change? such as the gloves

Archimus12 committed:
Organises Drobes

Organises most drobes in this order:
Sort order: Hud, Accessories, pens/crayons, Hat, Neck, Mask, bais suit, basic skirt, extra clothes, coat, gloves, shoes, backpack, satchel, duffel, belts, bags, other items then Headset.
Adds sec headsets (not bowman) to DetDrobe

Archimus12 commented:
I have now organised most drobes in this order:
Sort order: Hud, Accessories, pens/crayons, Hat, Neck, Mask, bais suit, basic skirt, extra clothes, coat, gloves, shoes, backpack, satchel, duffel, belts, bags, other items then Headset.
I also added sec headsets (not bowman) to DetDrobe

Archimus12 committed:
DetDrobe organising

Archimus12 committed:
Organises Winter Coats in Drobes

Moves winter coats below over over coats such as lab coats and hazard vests

Dorsisdwarf commented:
I thought the autodrobes were for cosmetic gear (replacing the crowded cosmetic wardrobes every department was full of before) and departmental vendors were where you're supposed to get actual kit?

Archimus12 commented:
Not every department has a vendor, only sec and med do? Also HUDs are glasses which are clothes.

I am fine with removing HUDs from the pr but that brings up the RoboDrobe HUDs that are already in game.

Mappers sometimes forgot to place semi important clothing (such as science goggles and gloves in science) so these changes would ensure that people have the basic clothing for the job.

On Terry, shift start health HUDs are often stolen due to being on a table in the treatment area.

Drobes are currently very inconsistent contents wise, cargo drobes have mail bags whereas science drobes do not have science bags, RoboDrobe has HUDs whereas others do not, and also MortiDrobe (coroner) contains non clothing items such as body bag boxes, coroner med kits, and the autopsy scanner (also table clock and ritual knife when hacked)

Sealed101 commented:
huds are early tech and are easy to print but they are forgotten most of the time unless miners scream for nvg mesons. 

no hurt in having a few huds or gloves in a vendor that's more consistent in mapping usage. huds and medical gloves of both types are not items that could be abused. some maps lack mapped-in gloves or job-specific huds, and this will inevitably be the case, so making precautions for that plus making those items pricier for non-departmental players would work out fine i think. 

as for the diag huds, those are limited to 2, which is the current limit of roboticist slots, and the robovend is basically an equipment locker, with the caveat of meme robes hidden behind contraband. it's not an example to follow by really, but the items in there are to some extent vital to the job, so i don't see why would this not be plausible for other jobs

thanks hans mothman

optimumtact commented:
>no hurt in having a few huds or gloves in a vendor that's more consistent in mapping usage

this is exactly why nobody prints them

Archimus12 committed:
Removes HUDs

Removes the HUDs, leaves the diagnostic ones though

Archimus12 commented:
I have now removed the HUDs, I have left the diagnostic ones in though as they were already there before the pr.

Archimus12 committed:
Undoes adding sec headsets to drobes

Archimus12 commented:
I undid adding sec headsets to DetDrobes and SecDrobes, Det drobe is already really full and SecDrobes in checkpoints can be looted.

JohnFulpWillard commented on code/modules/vending/wardrobes.dm:
```diff
@@ -51,32 +51,34 @@
+		/obj/item/storage/backpack/duffelbag/med = 4,
+		/obj/item/storage/backpack/medic = 4,
+		/obj/item/storage/backpack/satchel/med = 4,
+		/obj/item/radio/headset/headset_med = 4,
```
Headsets aren't in the vendor because they're in the closets.

JohnFulpWillard commented on code/modules/vending/wardrobes.dm:
```diff
@@ -92,19 +94,20 @@
+		/obj/item/storage/backpack/industrial = 3,
+		/obj/item/storage/backpack/satchel/eng = 3,
+		/obj/item/storage/backpack/duffelbag/engineering = 3,
+		/obj/item/radio/headset/headset_eng = 3,
```
Headsets aren't here because they're in the closets.

JohnFulpWillard commented on code/modules/vending/wardrobes.dm:
```diff
@@ -202,18 +212,21 @@
+		/obj/item/storage/backpack/science = 3,
+		/obj/item/storage/backpack/satchel/science = 3,
+		/obj/item/storage/backpack/duffelbag/science = 3,
+		/obj/item/storage/bag/xeno = 3,
```
These aren't here because they're in the closet.

Archimus12 committed:
Removes med and engie headsets as well as xenobio bag

Archimus12 commented:
I have now removed the med and engie headsets due to them being in lockers as well as the science xenobio bag as it is in lockers.

Archimus12 committed:
Removes viro bio bag and atmos headsets due to them being in lockers

Archimus12 commented:
I have removed viro bio bag and atmos headsets due to them being in lockers

Archimus12 committed:
Revert "Removes viro bio bag and atmos headsets due to them being in lockers"

This reverts commit 4faf96f7bcef1278e1e26b0898bfff72add1201e.

Archimus12 committed:
Removes viro bio bag and atmos headsets due to them being in lockers without declaring myself as the host

Archimus12 commented:
accidently forgot to untick admins.txt, now I have removed viro bio bag and atmos headsets due to them being in lockers without declaring myself as the host

ExcessiveUseOfCobblestone commented on code/modules/vending/wardrobes.dm:
```diff
@@ -284,22 +294,21 @@
-		/obj/item/storage/medkit/coroner = 1,
 		/obj/item/toy/crayon/white = 2,
-		/obj/item/clothing/mask/surgical = 1,
-		/obj/item/clothing/shoes/sneakers/black = 1,
```
the sneakers are no long available, was this intentional?

Archimus12 replied:
no it wasn't, I will fix that

ExcessiveUseOfCobblestone commented:
> > no hurt in having a few huds or gloves in a vendor that's more consistent in mapping usage
> 
> this is exactly why nobody prints them

should be removed from maps outside of vendors so people have to pay for them if they want them.

Archimus12 committed:
Re adds coroner shoes

Archimus12 commented:
I have added the coroner black shoes back

ExcessiveUseOfCobblestone commented:
whats the general reasoning behind closets vs. vendors, is it because these items are meant to kit people who are hop'd into the job without having to pay?

JohnFulpWillard commented:
It's put in closets when available, and put in vending machines otherwise. Medical, Engineering and Security are the only departments with job closets, so theyre the 3 departments with radio in their respective closets instead of the vendors.

ExcessiveUseOfCobblestone commented:
I guess my question is why not just stuff everything in a vendor since they have a UI?

YakumoChen commented:
Why can't we just give some of these jobs a closet? Chemistry already has tons of space and 2-3 related closets, for example.

Archimus12 commented:
> It's put in closets when available, and put in vending machines otherwise. Medical, Engineering and Security are the only departments with job closets, so theyre the 3 departments with radio in their respective closets instead of the vendors.
> 
> However this isn't a perfect system as it's inconsistent, and leaves some jobs (Paramedic, Chemist, Coroner) without a way to replace their headsets except begging their department to open a closet.

Coroner already has a service med headset in their drobe so I am leaving it in their but I have removed my headset additions to med, engi, sec and atmos drobes due to their headsets being in lockers, I currently do not know how to do mapping so am not sure how to add more job closets but I could try to learn mapping and add more job closets with related headsets and equipment.

JohnFulpWillard reviewed (approved):
Service/Science jobs don't have closets so all their headsets are in their vendors which is a good way to do it I think. Medical/Engineering don't follow this which is a shame. Not sure if this is something we should standardize in the future.

Archimus12 commented:
> Service/Science jobs don't have closets so all their headsets are in their vendors which is a good way to do it I think. Medical/Engineering don't follow this which is a shame. Not sure if this is something we should standardize in the future.

should I split this PR into just a few drobe clothe additions and organising drobes and make another pr maybe with a coding project post on the Discord that goes into making consistency for headsets in drobes vs lockers?
